tests/beaker_tests/README-beaker-script-ref.md

```diff
@@ -0,0 +1,232 @@
+# Beaker Test Script Guide to Test Inputs
```

Comment:
FWIW I've got a lot of regrets about putting the YAML readme in cmd_ref/ instead of doc/, so let's not try to continue that anti-pattern. :-)